beacond/cmd/types.go

```diff
@@ -63,14 +62,6 @@ import (
 /* -------------------------------------------------------------------------- */
 
 type (
-	// ABCIMiddleware is a type alias for the ABCIMiddleware.
-	ABCIMiddleware = middleware.ABCIMiddleware[
-		*BeaconBlock,
-		*BlobSidecars,
-		*Genesis,
-		*SlotData,
-	]
-
```

Comment:
I believe unnecessary once Middleware has been encapsulated in consensus service